### DISCUSSION TITLE
Have to Clean solution and run again for changes in XAML

### DISCUSSION THREAD

**Discussion by naarsi** [unoplatform/uno#16012] [Q&A] [answered]
XAML changes won't be applied on hot reload. Every time I make some change in the XAML, I have to clean the solution and run it again. 
Avalonia and WinUI hot reloads on XAML changes, same VS version.
Please let me know if you need any logs.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report, this is a preview feature in VS that is enabled by default, see https://github.com/unoplatform/uno/discussions/15807#discussioncomment-8715083. We'll be addressing this issue in the upcoming uno release.

**Reply by naarsi** [reply]
Thanks for the info. 

However, I have created a new solution (with different options in the UNO Wizard) just to re-verify and the new solution hot reloads on any XAML changes.

I'm not sure if this helps but:
I have opted out Uno Navigation & Uno Toolkit in the new solution and XAML changes are immediately reflected. 
Had them in the old solution where latest XAML changes doesn't reflect unless I clean the solution.
